# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/gcc41-arm.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Due raise needed now in libgcc.

  - Rene Rebe <rene@exactcode.de>

--- ./diet.c.vanilla	2006-03-27 22:27:51.066417750 +0200
+++ ./diet.c	2006-03-27 22:28:36.837278250 +0200
@@ -403,7 +403,9 @@
       if (_link) {
 	if (profile) *dest++="-lgmon";
 	*dest++=c; *dest++=(char*)libgcc;
-	if (!strcmp(shortplatform,"sparc") || !strcmp(shortplatform,"sparc64")) {
+	if (!strcmp(shortplatform,"sparc") ||
+	    !strcmp(shortplatform,"sparc64") ||
+	    !strcmp(shortplatform,"arm") ) {
 	  *dest++=c;
 	}
       }
